### ISSUE TITLE
[video_player] VideoPlayer & ClipRRect doesn't work with Skia (Crash)

### ISSUE THREAD

**Issue by nilsreichardt** [flutter/flutter#56785]
## Steps to Reproduce
1. Add video_player to your project
2. Copy this code:
```
import 'package:video_player/video_player.dart';
import 'package:flutter/material.dart';

void main() => runApp(VideoApp());

class VideoApp extends StatefulWidget {
  @override
  _VideoAppState createState() => _VideoAppState();
}

class _VideoAppState extends State<VideoApp> {
  VideoPlayerController _controller;

  @override
  void initState() {
    super.initState();
    _controller = VideoPlayerController.network(
        'https://sharezone.net/meme-submission-too-late')
      ..initialize().then((_) {
        setState(() {});
      });
  }

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Video Demo',
      home: Scaffold(
        body: Center(
          child: _controller.value.initialized
              ? ClipRRect(
                  borderRadius: BorderRadius.all(Radius.circular(10)),
                  child: AspectRatio(
                    aspectRatio: _controller.value.aspectRatio,
                    child: VideoPlayer(_controller),
                  ),
                )
              : CircularProgressIndicator(),
        ),
      ),
    );
  }

  @override
  void dispose() {
    super.dispose();
    _controller.dispose();
  }
}

``` 
3. Run `flutter run -d chrome --release --dart-define=FLUTTER_WEB_USE_SKIA=true`

Console is clear (no errors).

Without `ClipRRect` it works.

**Comment by TahaTesser** [comment]
<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.19.0-0.0.pre, on Mac OS X 10.15.4 19E287, locale
    en-GB)
    • Flutter version 1.19.0-0.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision a849daf283 (3 days ago), 2020-05-07 18:59:02 -0700
    • Engine revision 3953c3ccd1
    • Dart version 2.9.0 (build 2.9.0-5.0.dev 4da5b40fb6)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/SDK
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/SDK
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.45.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.10.1

[✓] Connected device (3 available)
    • macOS      • macOS      • darwin-x64     • Mac OS X 10.15.4 19E287
    • Web Server • web-server • web-javascript • Flutter Tools
    • Chrome     • chrome     • web-javascript • Google Chrome 81.0.4044.138

• No issues found!
tahatesser@Tahas-Ma
``` 

</details>

**Comment by TahaTesser** [comment]
I can reproduce it on `Channel beta, 1.25.0-8.3.pre`, however, it seems to be fixed in `Channel dev, 1.26.0-12.0.pre` and `Channel master, 1.26.0-13.0.pre.120` when running in canvaskit. 



<details>
<summary>code sample</summary>

```dart
import 'package:video_player/video_player.dart';
import 'package:flutter/material.dart';

void main() => runApp(VideoApp());

class VideoApp extends StatefulWidget {
  @override
  _VideoAppState createState() => _VideoAppState();
}

class _VideoAppState extends State<VideoApp> {
  VideoPlayerController _controller;

  @override
  void initState() {
    super.initState();
    _controller = VideoPlayerController.network(
        'https://sharezone.net/meme-submission-too-late')
      ..initialize().then((_) {
        setState(() {});
      });
  }

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Video Demo',
      home: Scaffold(
        body: Center(
          child: _controller.value.initialized
              ? ClipRRect(
                  borderRadius: BorderRadius.all(Radius.circular(10)),
                  child: AspectRatio(
                    aspectRatio: _controller.value.aspectRatio,
                    child: VideoPlayer(_controller),
                  ),
                )
              : CircularProgressIndicator(),
        ),
      ),
    );
  }

  @override
  void dispose() {
    super.dispose();
    _controller.dispose();
  }
}
```
</details>

<details>
<summary>logs</summary>

```bash
[  +87 ms] executing: [/Users/tahatesser/Code/flutter_beta/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +63 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 5d36f2e7f5387b6c751449258ade8e4e6edf99be
[   +1 ms] executing: [/Users/tahatesser/Code/flutter_beta/] git tag --points-at 5d36f2e7f5387b6c751449258ade8e4e6edf99be
[ +107 ms] Exit code 0 from: git tag --points-at 5d36f2e7f5387b6c751449258ade8e4e6edf99be
[   +1 ms] 1.25.0-8.3.pre
[  +65 ms] executing: [/Users/tahatesser/Code/flutter_beta/] git rev-parse --abbrev-ref --symbolic @{u}
[  +12 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/beta
[        ] executing: [/Users/tahatesser/Code/flutter_beta/] git ls-remote --get-url origin
[  +20 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +67 ms] executing: [/Users/tahatesser/Code/flutter_beta/] git rev-parse --abbrev-ref HEAD
[  +15 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] beta
[   +7 ms] executing: sw_vers -productName
[  +17 ms] Exit code 0 from: sw_vers -productName
[        ] macOS
[        ] executing: sw_vers -productVersion
[  +16 ms] Exit code 0 from: sw_vers -productVersion
[   +1 ms] 11.1
[        ] executing: sw_vers -buildVersion
[  +16 ms] Exit code 0 from: sw_vers -buildVersion
[        ] 20C69
[   +8 ms] executing: sysctl hw.optional.arm64
[   +6 ms] Exit code 1 from: sysctl hw.optional.arm64
[        ] sysctl: unknown oid 'hw.optional.arm64'
[  +72 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +73 ms] executing: /Volumes/Extreme/SDK/platform-tools/adb devices -l
[  +10 ms] executing: /usr/bin/xcode-select --print-path
[   +8 ms] Exit code 0 from: /usr/bin/xcode-select --print-path
[        ] /Volumes/Extreme/Xcode.app/Contents/Developer
[   +8 ms] executing: sysctl hw.optional.arm64
[   +6 ms] Exit code 1 from: sysctl hw.optional.arm64
[        ] sysctl: unknown oid 'hw.optional.arm64'
[        ] executing: xcrun xcodebuild -version
[ +336 ms] Exit code 0 from: xcrun xcodebuild -version
[        ] Xcode 12.3
           Build version 12C33
[   +3 ms] executing: xcrun xcdevice list --timeout 2
[   +4 ms] xcrun simctl list --json devices
[        ] executing: xcrun simctl list --json devices
[  +12 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +67 ms] executing: /Volumes/Extreme/SDK/platform-tools/adb devices -l
[        ] executing: xcrun xcdevice list --timeout 2
[        ] xcrun simctl list --json devices
[        ] executing: xcrun simctl list --json devices
[  +23 ms] Running "flutter pub get" in video_player_example...
[   +5 ms] executing: [/Users/tahatesser/AndroidStudioProjects/triage-examples/flutter_plugin_examples/video_player_example/] /Users/tahatesser/Code/flutter_beta/bin/cache/dart-sdk/bin/pub --verbose get --no-precompile
[  +62 ms] List of devices attached
[   +5 ms] List of devices attached
[  +55 ms] {
             "devices" : {
               "com.apple.CoreSimulator.SimRuntime.tvOS-14-3" : [

               ],
               "com.apple.CoreSimulator.SimRuntime.watchOS-7-2" : [

               ],
               "com.apple.CoreSimulator.SimRuntime.iOS-14-3" : [
                 {
                   "dataPath" : "\/Users\/tahatesser\/Library\/Developer\/CoreSimulator\/Devices\/DC25925E-876C-4CFE-A2F7-0640782D22A5\/data",
                   "logPath" : "\/Users\/tahatesser\/Library\/Logs\/CoreSimulator\/DC25925E-876C-4CFE-A2F7-0640782D22A5",
                   "udid" : "DC25925E-876C-4CFE-A2F7-0640782D22A5",
                   "isAvailable" : true,
                   "deviceTypeIdentifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-12-Pro-Max",
                   "state" : "Shutdown",
                   "name" : "iPhone 12 Pro Max"
                 }
               ]
             }
           }
[  +22 ms] {
             "devices" : {
               "com.apple.CoreSimulator.SimRuntime.tvOS-14-3" : [

               ],
               "com.apple.CoreSimulator.SimRuntime.watchOS-7-2" : [

               ],
               "com.apple.CoreSimulator.SimRuntime.iOS-14-3" : [
                 {
                   "dataPath" : "\/Users\/tahatesser\/Library\/Developer\/CoreSimulator\/Devices\/DC25925E-876C-4CFE-A2F7-0640782D22A5\/data",
                   "logPath" : "\/Users\/tahatesser\/Library\/Logs\/CoreSimulator\/DC25925E-876C-4CFE-A2F7-0640782D22A5",
                   "udid" : "DC25925E-876C-4CFE-A2F7-0640782D22A5",
                   "isAvailable" : true,
                   "deviceTypeIdentifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-12-Pro-Max",
                   "state" : "Shutdown",
                   "name" : "iPhone 12 Pro Max"
                 }
               ]
             }
           }
[  +46 ms] MSG : Resolving dependencies...
[   +1 ms] MSG : Got dependencies!
[  +41 ms] Running "flutter pub get" in video_player_example... (completed in 806ms)
[  +54 ms] Found plugin video_player at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player-1.0.1/
[  +10 ms] Found plugin video_player_web at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player_web-0.1.4/
[ +125 ms] Found plugin video_player at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player-1.0.1/
[   +3 ms] Found plugin video_player_web at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player_web-0.1.4/
[  +19 ms] Generating /Users/tahatesser/AndroidStudioProjects/triage-examples/flutter_plugin_examples/video_player_example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[ +291 ms] Launching lib/main.dart on Chrome in debug mode...
[ +116 ms] Updating assets
[ +100 ms] Waiting for connection from debug service on Chrome...
[  +18 ms] Found plugin video_player at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player-1.0.1/
[   +1 ms] Found plugin video_player_web at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player_web-0.1.4/
[  +21 ms] Found plugin video_player at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player-1.0.1/
[   +2 ms] Found plugin video_player_web at /Users/tahatesser/.pub-cache/hosted/pub.dartlang.org/video_player_web-0.1.4/
[  +45 ms] <- reset
[   +7 ms] /Users/tahatesser/Code/flutter_beta/bin/cache/dart-sdk/bin/dart --disable-dart-dev /Users/tahatesser/Code/flutter_beta/bin/cache/artifacts/engine/darwin-x64/frontend_server.dart.snapshot --sdk-root /Users/tahatesser/Code/flutter_beta/bin/cache/flutter_web_sdk/ --incremental --target=dartdevc --debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_USE_SKIA=true --output-dill /var/folders/37/jzy2jdc16sb3p89cn_6g967c0000gn/T/flutter_tools.gxanHq/flutter_tool.0fDfuU/app.dill --libraries-spec file:///Users/tahatesser/Code/flutter_beta/bin/cache/flutter_web_sdk/libraries.json --packages /Users/tahatesser/AndroidStudioProjects/triage-examples/flutter_plugin_examples/video_player_example/.dart_tool/package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-root /var/folders/37/jzy2jdc16sb3p89cn_6g967c0000gn/T/flutter_tools.gxanHq/flutter_tools.RDc9R9 --filesystem-scheme org-dartlang-app --initialize-from-dill build/3b22c16eb48c043f81099103590514a0.cache.dill.track.dill --platform file:///Users/tahatesser/Code/flutter_beta/bin/cache/flutter_web_sdk/kernel/flutter_ddc_sdk.dill --no-sound-null-safety
[  +15 ms] <- compile org-dartlang-app:/web_entrypoint.dart
[+2389 ms] [
             {
               "simulator" : true,
               "operatingSystemVersion" : "14.3 (18C61)",
               "available" : true,
               "platform" : "com.apple.platform.iphonesimulator",
               "modelCode" : "iPhone13,4",
               "identifier" : "DC25925E-876C-4CFE-A2F7-0640782D22A5",
               "architecture" : "x86_64",
               "modelUTI" : "com.apple.iphone-12-pro-max-1",
               "modelName" : "iPhone 12 Pro Max",
               "name" : "iPhone 12 Pro Max"
             },
             {
               "simulator" : false,
               "operatingSystemVersion" : "14.3 (18C66)",
               "interface" : "usb",
               "available" : true,
               "platform" : "com.apple.platform.iphoneos",
               "modelCode" : "iPhone11,8",
               "identifier" : "00008020-001059882212002E",
               "architecture" : "arm64e",
               "modelUTI" : "com.apple.iphone-xr-1",
               "modelName" : "iPhone XR",
               "name" : "Taha’s iPhone"
             }
           ]
[   +5 ms] [
             {
               "simulator" : true,
               "operatingSystemVersion" : "14.3 (18C61)",
               "available" : true,
               "platform" : "com.apple.platform.iphonesimulator",
               "modelCode" : "iPhone13,4",
               "identifier" : "DC25925E-876C-4CFE-A2F7-0640782D22A5",
               "architecture" : "x86_64",
               "modelUTI" : "com.apple.iphone-12-pro-max-1",
               "modelName" : "iPhone 12 Pro Max",
               "name" : "iPhone 12 Pro Max"
             },
             {
               "simulator" : false,
               "operatingSystemVersion" : "14.3 (18C66)",
               "interface" : "usb",
               "available" : true,
               "platform" : "com.apple.platform.iphoneos",
               "modelCode" : "iPhone11,8",
               "identifier" : "00008020-001059882212002E",
               "architecture" : "arm64e",
               "modelUTI" : "com.apple.iphone-xr-1",
               "modelName" : "iPhone XR",
               "name" : "Taha’s iPhone"
             }
           ]
[+12841 ms] Waiting for connection from debug service on Chrome... (completed in 15.3s)
[        ] Synced 28.9MB.
[        ] <- accept
[        ] Caching compiled dill
[ +273 ms] Using Google Chrome 88.0.4324.96 

[+3002 ms] [CHROME]:
[   +1 ms] [CHROME]:DevTools listening on ws://127.0.0.1:58424/devtools/browser/1f57deef-3fd8-4594-8b07-5b338c0d8dee
[ +905 ms] DwdsInjector: Received request for entrypoint at http://localhost:58411/main_module.bootstrap.js
[   +7 ms] MetadataProvider: Loading debug metadata...
[  +18 ms] MetadataProvider: Loaded debug metadata
[   +9 ms] DwdsInjector: Injected debugging metadata for entrypoint at http://localhost:58411/main_module.bootstrap.js
[+2426 ms] DevHandler: Debug service listening on ws://127.0.0.1:58487/LqtcQFJ-SCY=/ws

[   +9 ms] Debug service listening on ws://127.0.0.1:58487/LqtcQFJ-SCY=/ws
[        ] Running with unsound null safety
[        ] For more information see https://dart.dev/null-safety/unsound-null-safety
[   +1 ms] Warning: Flutter's support for web development is not stable yet and hasn't
[        ] been thoroughly tested in production environments.
[        ] For more information see https://flutter.dev/web
[        ] 🔥  To hot restart changes while running, press "r" or "R".
[        ] For a more detailed help message, press "h". To quit, press "q".
```
</details>


<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel beta, 1.25.0-8.3.pre, on macOS 11.1 20C69 darwin-x64, locale en-GB)
    • Flutter version 1.25.0-8.3.pre at /Users/tahatesser/Code/flutter_beta
    • Framework revision 5d36f2e7f5 (8 days ago), 2021-01-14 15:57:49 -0800
    • Engine revision 7a8f8ca02c
    • Dart version 2.12.0 (build 2.12.0-133.7.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (2 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 14.3
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 88.0.4324.96

• No issues found!
```
```bash
[✓] Flutter (Channel dev, 1.26.0-12.0.pre, on macOS 11.1 20C69 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-12.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision a706cd2112 (8 days ago), 2021-01-14 18:20:26 -0500
    • Engine revision effb529ece
    • Dart version 2.12.0 (build 2.12.0-224.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (3 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 14.3
    • macOS (desktop)        • macos                     • darwin-x64     • macOS 11.1 20C69 darwin-x64
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 88.0.4324.96

• No issues found!
```
```bash
[✓] Flutter (Channel master, 1.26.0-13.0.pre.120, on macOS 11.1 20C69 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-13.0.pre.120 at /Users/tahatesser/Code/flutter_master
    • Framework revision 46b0797b58 (5 hours ago), 2021-01-22 14:48:49 +0500
    • Engine revision 90641fa923
    • Dart version 2.12.0 (build 2.12.0-248.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (3 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 14.3
    • macOS (desktop)        • macos                     • darwin-x64     • macOS 11.1 20C69 darwin-x64
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 88.0.4324.96

• No issues found!
```

</details>


Thank you

**Comment by hterkelsen** [comment]
Confirmed that this works correctly on latest master.
